addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Tankerkoenig Binding
+Bundle-SymbolicName: org.openhab.binding.tankerkoenig;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.collect,
+ com.google.gson,
+ org.apache.commons.io;version="2.2.0",
```

Comment:
Do you really need that specific version of commons-io? If not, please remove version constraint.